### PR TITLE
Added description to CustomFields argument

### PR DESCRIPTION
**Pull request by dem-evgeny**

### CODE REVIEW

**Comment by dem-evgeny** [comment]
Fixes: https://github.com/demisto/etc/issues/10520

**Comment by dem-evgeny** [comment]
@gshriki A little bit problematic because if for example the name itself will have u_ so it could have u_u_blabla...

File: Integrations/integration-ServiceNow.yml [hunk 1]
```diff
@@ -828,7 +828,8 @@ script:
     - name: template
       description: Template name to use as a base to create new incidents.
     - name: custom_fields
-      description: 'Custom fields in this format: fieldname=value;fieldname=value...'
+      description: 'Custom fields in this format: fieldname=value;fieldname=value...
+        (You don''t need to add "u_" in the name of the custom field. It adds it automatically.'
```
Comment (anara123) [inline Integrations/integration-ServiceNow.yml]: shouldn't `don''t`  be `don\'t`?

Comment (dem-evgeny) [reply Integrations/integration-ServiceNow.yml]: I wrote 
You don't need to add "u_" in the name of the custom field. It adds it automatically.
Demisto added the duplicate ''

**Comment by gshriki** [comment]
ok. so why would the user add it? where does he see it?

**Comment by dem-evgeny** [comment]
@gshriki If it is a power user that is looking in the JSON and not in the GUI. Only in the JSON it adds "u_". I had some user like this recently.

**Comment by gshriki** [comment]
Agreed not to merge

**Pull request closed without merging**